core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3325,7 +3326,13 @@ public String toString() {
   public static final PropertyKey WORKER_TIERED_STORE_LEVEL0_DIRS_QUOTA =
       new Builder(Template.WORKER_TIERED_STORE_LEVEL_DIRS_QUOTA, 0)
           .setDefaultValue(String.format("${%s}", Name.WORKER_RAMDISK_SIZE))
-          .setDescription("The capacity of the top storage tier.")
+          .setDescription(String.format(
+              "A comma-separated list of capacities (e.g., \"500MB,500MB,5GB\") for each storage "
+                  + "directory on the top storage tier specified by %s. "
+                  + "For any \"MEM\"-type media (i.e, the ramdisks), this value should be set "
+                  + "equivalent to the value specified by %s.",
+              PropertyKey.WORKER_TIERED_STORE_LEVEL0_DIRS_PATH.mName,
+              PropertyKey.WORKER_RAMDISK_SIZE.mName))
```

Comment:
Prefer `Name.WORKER_RAMDISK_SIZE` here instead since I believe that what we use elsewhere. the `LEVEL0_DIRS_PATH` PK doesn't have a `Name.*` because it's a template property so that one is ok